### ISSUE TITLE
Back in time (root) (1.3.3 on Linux Mint 20.03) Font too small... how to enlarge ?

### ISSUE THREAD

**Issue by guy6308** [bit-team/backintime#1475]
I installed  backintime-qt 1.2.1 from the software app in Linux Mint 20.03

When I run the (root) version the font is incredibly small.

Is there a way to make it bigger ??

Thank you.

**Comment by buhtz** [comment]
Dear guy6308,
thanks for reporting.

Do other applications have the same problem? Can you provide a screenshot of Back In time with another application side-by-side to compare the font sizes.

Please be aware that 1.2.1 was release 4 years ago. You should try a newer version. The latest is 1.3.3.

I don't know much about Linux Mint but it seems to be Debian-based. So it might be possible to use the PPA which is [described here](https://github.com/bit-team/backintime#alternative-installation-options).

Another way would be to ask the Mint maintainers how to deal with this situation. It seems they offer Mint 20.03 as a long term version until 2025. But we as upstream maintainers can not support old versions that long.

If you are able to reproduce the problem with a newer version please report back.

**Comment by guy6308** [comment]
Hello,
Following your advice, I installed (from the ppa) version 1.3.3-3
Unfornutalely it does not solve the problem.  However I found something : when I installed backintime ( 1.3.3-3 ) on another linux mint 20.3 (exact same OS), at first the backintime non root ehibited the same problem as the backintime root UNTIL I ran Qt5 Configuration tool (qt5ct) to set the font size to an acceptable 14pt instead of the 10pt set by default.

Obviously, I tried to run the qt5ct utility as root  hoping that would also solve the problem for backintime root, but to no avail, it would not run.

I an including here the screenshots you requested, namely backintime root and non root on same screen and then backintime root and "synaptic" ( a well-known application in the Ubuntu/Mint/debian world).
Thanks a lot for your help!

Guy

![Capture d’écran du 2023-07-13 21-34-27](https://github.com/bit-team/backintime/assets/139502159/8c3d9e69-2e2e-407b-9458-ea74f270b5b6)
![Capture d’écran du 2023-07-13 21-35-29](https://github.com/bit-team/backintime/assets/139502159/039b10b8-9f90-490f-9ad5-6f2d2b759270)

**Comment by aryoda** [comment]
> ... UNTIL I ran Qt5 Configuration tool (qt5ct) to set the font size to an acceptable 14pt instead of the 10pt set by default.

Yes, if you are using Qt5ct to configure the Qt5 styles this is the way to go

> Obviously, I tried to run the qt5ct utility as root hoping that would also solve the problem for backintime root, but to no avail, it would not run.

How did you try to run it?

I just tried `sudo qt5ct` in my terminal and it opened but showed me "The application is not configured correctly":

![image](https://github.com/bit-team/backintime/assets/11374410/9dd0a55b-87ff-49a2-918b-023db77d498c)

So somehow this env var must be set for the root user (I am searching...)

**Comment by aryoda** [comment]
I think to solve this you can add this line

`export QT_QPA_PLATFORMTHEME=qt5ct`

in `/etc/environment` (e.g. via `sudo gedit /etc/environment`).

If you should ever change the qt5 style to another one than `qt5ct` don't forget to remove this line or it will bother you by being still existent even after de-installing `qt5ct`

@guy6308 Could you please give us feed-back here if this helps. We should add this work-around to our FAQ then...

**Comment by guy6308** [comment]
YES YES!!!!!   Thank you a LOT Aryoda!!!!   It worked!    It never occured to me to just use sudo qt5ct... I was doing a su - and then try to run qt5ct that way. I even tried with pkexec.  I even tried to copy my ~/.config/qt5ct to /root ...
Thanks to you I understand now that one needs to set the QT_QPA_PLATFORMTHEME for root to read it always...

I am very grateful and my old strained eyes thank you a lot!

Guy L.

**Comment by aryoda** [comment]
Internal TODO(s) before closing this issue:

- Add an entry to our FAQ to mention this configuration step (it cannot be done automatically since it is point-in-time deliberate user decision to use `qt5ct` and could change at any time in the future so the user must do this)

**Comment by buhtz** [comment]
Dear @guy6308,
I do not understand all details here. So I am not able to provide a text for the FAQ.

Would you like to contribute to the project and provide a litte FAQ entry about your problem and how it can be solved?

You give the text here as an issue comment or edit the `FAQ.md` file and open a Pull Request.

**Comment by buhtz** [comment]
Hello guy6308,

I would like to close this issue. But I am not able to reproduce it nor understand all details about the solution.
If you could explain it somehow I could add this problem and its solution to the FAQ.

Regards,
Christian

**Comment by buhtz** [comment]
Hello @guy6308,
I am sorry, but without response I have no other choice to simply close the issue. It would help the project and other users if you would write down your way of solution.

Regards,
Christian

**Comment by buhtz** [comment]
Closing this ticket based on the comment above. Feel free to reopen
if the problem still exists. Thank you for your efforts. If you have
any further questions, ideas or encounter any other issues, please
don't hesitate to let us know.

Best regards,
Christian
